### PR TITLE
Add Consecutive_up and Consecutive_down for load_balancer_monitor res…

### PR DESCRIPTION
**Pull request by Nmishin**
## Description

We can set the consecutive_down and consecutive_up parameter values for the Cloudflare Load Balancer Monitor via [API](https://developers.cloudflare.com/api/operations/load-balancer-monitors-create-monitor ) but not via terraform.

[Issue from terraform-provider-cloudflare](https://github.com/cloudflare/terraform-provider-cloudflare/issues/1876)

## Has your change been tested?

Yes, relevant unit tests have been updated to handle the addition of this field. Additionally, changes were ran against the pre-existing test suite to verify that no regressions were introduced.

## Types of changes

What sort of change does your code introduce/modify?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] This change is using publicly documented in [cloudflare/api-schemas](https://github.com/cloudflare/api-schemas) 
      and relies on stable APIs.

[1]: https://help.github.com/articles/closing-issues-using-keywords/

### CODE REVIEW

File: load_balancing.go [hunk 1]
```diff
@@ -64,23 +64,25 @@ type LoadBalancerOriginSteering struct {
...
+	ModifiedOn       *time.Time          `json:"modified_on,omitempty"`
+	Type             string              `json:"type"`
+	Description      string              `json:"description"`
+	Method           string              `json:"method"`
+	Path             string              `json:"path"`
+	Header           map[string][]string `json:"header"`
+	Timeout          int                 `json:"timeout"`
+	Retries          int                 `json:"retries"`
+	Interval         int                 `json:"interval"`
+	Consecutive_up   int                 `json:"consecutive_up"`
```
Comment (jacobbednarz) [inline load_balancing.go]:
```suggestion
	ConsecutiveUp   int                 `json:"consecutive_up"`
```

File: load_balancing.go [hunk 2]
```diff
@@ -64,23 +64,25 @@ type LoadBalancerOriginSteering struct {
...
+	Consecutive_down int                 `json:"consecutive_down"`
```
Comment (jacobbednarz) [inline load_balancing.go]:
```suggestion
	ConsecutiveDown int                 `json:"consecutive_down"`
```

**Review by jacobbednarz — CHANGES_REQUESTED** [review]
thanks for the PR. looks good overall but we should use camel casing for the struct names.

**Comment by jacobbednarz** [comment]
cc @gofish @tc80

**Review by tc80 — APPROVED** [review]

**Pull request merged**